mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -47,13 +47,22 @@ var applyCmd = &cobra.Command{
 mesheryctl perf apply meshery-profile --flags
 
 // Execute a Performance test with creating a new performance profile
-mesheryctl perf apply --profile meshery-profile-new --url <url>
+mesheryctl perf apply --profile meshery-profile-new --url "https://google.com"
 
-// Run Performance test using SMP compatible test configuration
-mesheryctl perf apply -f <filepath>
+// Run Performance test using SMP compatible test configuration (if available)
```

Comment:
`(if available)` why are we adding this here? I didn't get it.

Reply:
Wait... isn't SMP configuration optional one? Normally users don't have SMP config file by default right? Or do they?

Reply:
Correct me if I'm wrong 😨

Reply:
It's not optional, it's kinda like a main feature. So I don't think we should promote this as optional.

Reply:
It's not optional, it's kinda like a main feature. So I don't think we should promote this as optional.

Reply:
2 Times the msg is printed? 🤔 Ok bro, I'll drop that "optional" keyword... Any other changes I need to do or need to be aware of?

Reply:
> 2 Times the msg is printed?

yeah, I don't know why!